docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+Just like the previous iterations, the survey was chopped into three sections: ["Your needs"](#your-needs), ["Your product"](#your-product), and ["About you"](#about-you).
+
+## Your needs
+
+### How would you feel if you could no longer use MUI?
+
+Fortunately, and similar to 2019 and 2020, over 93% of the respondents would be disappointed if they could no longer use MUI (a sum of _"Very disappointed"_ with _“Somewhat disappointed”_). A very slight 1% decrease from 2020 (94%) but still as encouraging as it can be. MUI’s team will continue to grow to keep you all even more satisfied.
+
+<img src="/static/blog/2021-survey/1.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Bar chart: 62.7% Very disappointed, 30.4% Somewhat disappointed, 6.9% Not disappointed." />
+
+<p class="blog-description">1567 out of 1589 answered</p>
+
+On the flip side though, the amount of "Very disappointed” alone fell by 10,3%, and the number of "Not disappointed" grew by 1,1%, in comparison with 2020. When choosing the last option, respondents were able to write a free-form answer to justify the choice. Based on the data, here are the main reasons we noticed and possible explanations for why these came to be:
+
+**The number of available alternatives**
+
+Lately, with every new sunrise, there's a new UI component library. Naturally, considering the amount of iteration each one does on the same problems, patterns start to become more and more established. It's common to see libraries with very similar implementations of components, styling solutions, theming, etc. It gets harder to find ways to differentiate, to find something that would make one's iteration more compelling. That's a main reason we do this survey yearly, to shed light into improvements opportunities. Now that we have uncovered a bunch of spaces to explore, and together with our steadily growing team, we hope to focus on more than one scope at once with the same level of attention to detail and quality we've strived to have.
+
+**The introduction of a paid product**
+
+A big challenge with open-source is that the proportion of value captured by maintainers is close to zero. The MIT licensed model enabled a lot of growth but it naturally has reached a limit. In order to push our mission as much as we believe it's possible, we needed to introduce a way to capture more value. This was made by making MUI X open-core, instead of open-source, meaning having a full-time team of maintainers for the open-source and providing you with the most outstanding set of advanced components.
```

Comment:
Maybe we should explain more what `open-core` means and emphasize that `DataGrid` is available under MIT license?
A lot of users tend to think that X components are commercial only (for example https://twitter.com/Bifrost60541029/status/1485648490902528004)

Reply:
```suggestion
A big challenge with open-source is that the proportion of value captured by maintainers is close to zero. The MIT licensed model enabled a lot of growth but it naturally has reached a limit. In order to push our mission as much as we believe it's possible, we needed to introduce a way to capture more value. [Open-core](https://en.wikipedia.org/wiki/Open-core_model) allows us to still offer an MIT license (free) version of the X components while charging for a part of them, the one which demands more specialized attention (including support). 
```

Makes sense! Removed a bit of unnecessary text and swapped it for a brief mention of why open-core was relevant (linking a handy Wikipedia) and emphasizing there's still an MIT version of it.

Reply:
Way to go!
I have rephrased it a bit, let me know what you think :)

> [Open-core](https://en.wikipedia.org/wiki/Open-core_model) allows us to still offer an MIT licensed (free) version of the X components while charging for additional features that require more attention (including support).